alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -354,7 +355,10 @@ protected function getLivePackagerUrl($serverNode, $streamFormat = null)
 			$entryId = $this->getDynamicAttributes()->getEntryId();
 		}
 		
-		$livePackagerUrl = "$livePackagerUrl/p/$partnerID/e/$entryId/" . $serverNode->getSegmentDurationUrlString($segmentDuration);
+		$livePackagerUrl = "$livePackagerUrl/p/$partnerID/e/$entryId/";
```

Comment:
Is there a chance in the future that we'll want both segmentDurationUrlString and sessionIdUrlString on the same url?
Isn't it always one or the other? seems like there should be one call and the relevant serverNode will handle it accordingly.

Reply:
You are correct the currently one is for the old live and the other for the new but can in the future be server that need them both? yes
we wonder if the wrap both as one function of get extraParams or something - then each server will call all his releated function - do you think that better?

Reply:
Better to leave the legacy code untouched I guess.